gprofiler/gprofiler_types.py

```diff
@@ -44,3 +44,15 @@ def nonnegative_integer(value_str: str) -> int:
     if value < 0:
         raise configargparse.ArgumentTypeError("invalid non-negative integer value: {!r}".format(value))
     return value
+
+
+def integer_range(min_range: int, max_range: int) -> Callable[[str], int]:
+    def integer_range_check(value_str: str) -> int:
+        value = int(value_str)
+        if value < min_range or value > max_range:
```

Comment:
No one would expect the max to be included, change to `value >= max_range`

like `range()`